src/dos/dev_con.h

```diff
@@ -75,6 +75,14 @@ void device_CON::ClearAnsi()
 	ansi.numberofarg = 0;
 }
 
+enum ControlCode : uint8_t {
```

Comment:
Would it be feasible to move this enum somewhere to `/include/*.h` and make `program_more_output.cpp` reuse it?

Reply:
Oh, nice - I didn't see that block of constants. Thanks for this heads up, @FeralChild64.

```
// ASCII control characters
constexpr char code_ctrl_c = 0x03; // end of text
constexpr char code_bs     = 0x08; // backspace
constexpr char code_lf     = 0x0a; // line feed
constexpr char code_ff     = 0x0c; // form feed
constexpr char code_cr     = 0x0d; // carriage return
constexpr char code_esc    = 0x1b;
constexpr char code_del    = 0x7f;
```

I guess it should go in an `ascii.h` file, but for now we don't need all ~200+ characters defined - we can start with this set of control codes, and add as needed?

Will get this added and re-use it in `program_more_output.cpp`

Reply:
Definitely - no need to have all the characters for now.

Reply:
Fixed; dropped them into an enum class `Ascii`, so we can use `Ascii::LineFeed`, etc. I tried 'ControlCode::LineFeed`, but `Ascii::` was a lot more intuitive/obvious. So went with that.